client/scripts/com/monsters/maproom_advanced/PopupInfoEnemy.as

```diff
@@ -764,18 +764,18 @@ package com.monsters.maproom_advanced
          GLOBAL._currentCell = this._cell;
          if(this._cell._base == 1)
          {
-            BASE.LoadBase(null,0,this._cell._baseID,GLOBAL.e_BASE_MODE.WMVIEW,false,EnumYardType.MAIN_YARD);
+            BASE.LoadBase(null,0,this._cell._baseID,GLOBAL.e_BASE_MODE.WMVIEW,false,EnumYardType.MAIN_YARD, this._cell._cell_id);
```

Comment:
Reminder: Remove any new client side logic, we should emulate the original server.